x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/key_insights_panel/alerts_triggered_tile/esql_query.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getAlertsTriggeredEsqlCount } from './esql_query';
+
+// Mock the helper functions
+jest.mock('../../../queries/helpers', () => ({
+  getPrivilegedMonitorUsersJoin: jest.fn(
+    (namespace: string) => `| JOIN privileged_users_${namespace} ON user.name`
+  ),
+}));
+
+describe('getAlertsTriggeredEsqlCount', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should return a valid ESQL query with alerts index', () => {
+    const namespace = 'test-namespace';
+    const alertsIndexName = '.alerts-security.alerts-test';
+
+    const result = getAlertsTriggeredEsqlCount(namespace, alertsIndexName);
+
+    expect(result).toContain('FROM .alerts-security.alerts-test METADATA _id, _index');
+    expect(result).toContain('| JOIN privileged_users_test-namespace ON user.name');
+    expect(result).toContain('| STATS COUNT(*)');
```

Comment:
One easier way to test the exact content of a string is to use `.toMatchInlineSnapshot(`